.circleci/config.yml

```diff
@@ -188,13 +193,29 @@ jobs:
           command: |
             make pg-demo-coord-build-app
 
-#  publish_release:
-#    executor: docker_minio
-#    steps:
-#      - run:
-#          name: Dummy step
-#          command: |
-#            echo "Should reach here before publishing anything with a tagged release"
+  publish_release:
+    executor: docker_builder
+    # TODO(frederic):
+    # - acquire GITHUB_TOKEN for onecrobot to push to homebrew-datamon repo
+    #
+    # > goreleaser needs access to github API, not only the git repo.
+    # > Hence ssh key is not sufficient and GITHUB_TOKEN is required.
+    steps:
+      - run:
+          name: Prepare github release
+          command: |
+            # self-installing goreleaser
+            curl -sL https://git.io/goreleaser > goreleaser
+            chmod u+x goreleaser
+            release_notes="./notes/NOTES.${CIRCLE_TAG}.md"
+            if [[ -n ${CIRCLE_TAG} && -f ${RELEASE_NOTES} ]] ; then
+              # use custom release notes
+              echo "Adding release notes ${release_notes}"
+              opts="--release-notes ${release_notes}"
+              # otherwise, leaves standard changelog from goreleaser
+              # (picks up all commits)
+            fi
+            ./goreleaser ${opts-""}
```

Comment:
although the `get()` with default metaphor is a useful placeholder to mention vis. shell scripting, `$opts` or `${opts}` is equivalent here since the default is `""` i.e. empty.